### DISCUSSION TITLE
Best practice for audit logging

### DISCUSSION THREAD

**Discussion by Jiawei0227** [rook/rook#9370] [Q&A]
Hello guys, I am wondering if there is a way to collect audit logs for ceph and rook operations?

**Comment by leseb** [comment]
Both audit and mon logs are logged on the same stream on the daemons's standard out. How about continuously reading the it?

**Reply by Jiawei0227** [reply]
I am using the `log_to_file`=true. But somehow the audit log and the cluster log are mixed together... I follow the doc here to try separate them, but it does not seem to work:

https://docs.ceph.com/en/latest/rados/troubleshooting/log-and-debug/#confval-mon_cluster_log_file

**Reply by Jiawei0227** [reply]
nvm, I think it worked.
